test/test_background_flux_divergence.jl

```diff
@@ -0,0 +1,78 @@
+using Oceananigans
+using Test
+using CUDA:has_cuda
+using NCDatasets
+
+"""
+run_with_background_fields(arch; with_background=true)
+Run a model with or without background fields and compare the two.
```

Comment:
```suggestion
    function run_with_background_fields(arch; with_background=true)
    
Run a model with or without background fields and compare the two.
```

Trivial but this is the format for a docstring if that's what is intended.